src/ProjectTemplates/Web.ProjectTemplates/content/WebApi-FSharp/Controllers/WeatherForecastController.fs

```diff
@@ -1,4 +1,4 @@
-﻿namespace Company.WebApplication1.Controllers
+namespace Company.WebApplication1.Controllers
```

Comment:
Can we revert this change. Likely just a line ending diff. 

Also, let's revert the `googletest` submodule update below.